### ISSUE TITLE
Using WFPlayer with ArtPlayer

### ISSUE THREAD

**Issue by NinjaCoderDotCom** [zhw2590582/WFPlayer#30]
Hi, @zhw2590582 

I am trying to implement wfplayer with Artplayer. Do you have any examples of both of your libraries working together?

**Comment by zhw2590582** [comment]
Yes [demo](https://artplayer.org/?libs=https%3A%2F%2Fcdn.jsdelivr.net%2Fnpm%2Fwfplayer%2Fdist%2Fwfplayer.js&code=const%20%24wrap%20%3D%20document.querySelector(%27.video-wrap%27)%3B%0Aconst%20%24waveform%20%3D%20document.createElement(%27div%27)%3B%0A%24waveform.style.height%20%3D%20%27150px%27%3B%0A%24waveform.style.width%20%3D%20%27100%25%27%3B%0A%24wrap.appendChild(%24waveform)%3B%0A%0Aconst%20art%20%3D%20new%20Artplayer(%7B%0A%09container%3A%20%27.artplayer-app%27%2C%0A%09url%3A%20%27%2Fassets%2Fsample%2Fvideo.mp4%27%2C%0A%7D)%3B%0A%0Aart.on(%27ready%27%2C%20()%20%3D%3E%20%7B%0A%09const%20wf%20%3D%20new%20WFPlayer(%7B%0A%09%09container%3A%20%24waveform%2C%0A%09%09mediaElement%3A%20art.template.%24video%2C%0A%09%09scrollable%3A%20true%2C%0A%09%7D)%3B%0A%0A%09%2F%2F%20wf.load(%27%2Fassets%2Fsample%2Fvideo.mp4%27)%3B%0A%09wf.load(%27%2Fassets%2Fsample%2Faudio.mp3%27)%3B%0A%7D)%3B)
